RecoTracker/TkMSParametrization/plugins/MultipleScatteringParametrisationMakerESProducer.cc

```diff
@@ -0,0 +1,47 @@
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterDescription.h"
+#include "RecoTracker/TkMSParametrization/interface/MultipleScatteringParametrisationMaker.h"
+#include "RecoTracker/Record/interface/NavigationSchoolRecord.h"
+
+#include "RecoTracker/TkDetLayers/interface/GeometricSearchTracker.h"
+#include "RecoTracker/Record/interface/TrackerRecoGeometryRecord.h"
+#include "MagneticField/Engine/interface/MagneticField.h"
+#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
+
+class MultipleScatteringParametrisationMakerESProducer : public edm::ESProducer {
+public:
+  MultipleScatteringParametrisationMakerESProducer(edm::ParameterSet const& iConfig);
+
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+  // Using NavigationSchoolRecord here sounds a bit confusing, that
+  // that depends only on TrackerRecoGeometryRecord and
+  // IdealMagneticFieldRecord
+  std::unique_ptr<MultipleScatteringParametrisationMaker> produce(const NavigationSchoolRecord& iRecord);
```

Comment:
Note that I blatantly reused the `NavigationSchoolRecord` for this ESProduct, as that record depends exactly on the two records (`TrackerRecoGeometryRecord` and `IdealMagneticFieldRecord`) needed for the `MultipleScatteringParametrisation`). If that is considered too confusing, I can create a new record (or put it to some other existing record like `CkfComponentsRecord`, but those may have smaller IOVs than the intersection of `TrackerRecoGeometryRecord` and `IdealMagneticFieldRecord`).